chat-widget/src/common/utils.ts

```diff
@@ -266,4 +267,62 @@ export const createTimer = (): ITimer => {
             return ms;
         }
     };
+};
+
+export const transformCustomContext = (customContext: any): string  => {
```

Comment:
Can we move this to a separate file? Doesn't seem to fit util standard

Reply:
Removed it. Moved to the scripting layer.